piplmesh/account/views.py

```
@@ -75,4 +79,49 @@ def logout(request):
         url = request.POST.get(auth.REDIRECT_FIELD_NAME)
         return auth_views.logout_then_login(request, url)
     else:
-        raise exceptions.PermissionDenied
\ No newline at end of file
+        raise exceptions.PermissionDenied
+
+class PushView(generic_views.TemplateView):
```

Views or signals? In views you just raise exception and with `DEBUG` set to `True` it shows you technical response.

In signals exceptions are not propagated (I configured like that in case of passthrough view).